data/core/macros/abilities.cfg

```diff
@@ -198,9 +198,9 @@ This ability will not cure an affected unit of poison, however, only delay its e
         cumulative=no
         name= _ "leadership"
         female_name= _ "female^leadership"
-        description= _ "This unit can lead your own units that are next to it, making them fight better.
+        description= _ "This unit can lead your troops in battle.
 
-Adjacent own units of lower level will do more damage in battle. When a unit adjacent to, of a lower level than, and on the same side as a unit with Leadership engages in combat, its attacks do 25% more damage times the difference in their levels."
+All your adjacent troops of a lower level deal 25% more damage times the difference of their levels."
```

Comment:
Alright, how does this sound:
"Units with leadership increase the damage of each player-owned adjacent lower-level unit by 25% for each difference in level."
Which sounds better for the last clause "for each difference in level" or "times the difference of their levels"?

Reply:
That might be a good touch. 

I wonder if "a unit with leadership increases" is a bit better, so  both the leader unit and the underling unit will be singular and "their" in "times the difference of their levels" will clearly relate to the only possible pair in the context of this phrase: the difference of the leader and the underling levels.